src/lib/config.ts

```diff
@@ -509,6 +509,7 @@ export class default_config {
         "mkt!": "mktridactylrc -f",
         "mktridactylrc!": "mktridactylrc -f",
         mpvsafe: "js -p tri.excmds.shellescape(JS_ARG).then(url => tri.excmds.exclaim_quiet('mpv ' + url))",
+        exto: "extoptions"
```

Comment:
```suggestion
        exto: "extoptions",
        extpreferences: "extoptions",
        extp: "extpreferences",
```

Reply:
Thank  you for pointing this out. Fixed.